lib/server/server.js

```diff
@@ -559,7 +559,7 @@ function execute(port) {
   gaze(
     [
       '../docs/**/*', // docs
-      './**/*', // website
+      '*/*.*', // website
```

Comment:
On second thought, maybe this approach is flawed re: https://stackoverflow.com/a/21835063/2628398. If some file was in root of website it wouldn't be watched.

Off to sleep, will revisit this tomorrow...

Reply:
One last thing, it's confusing that `**/*.*` doesn't work...

Reply:
`.*` specifies any extension and is redundant because `*` would do the same (in most cases).

`*/*.*` only looks at the files directly under `website`, files which are deeper such as `blog` would not be watched.

I think `./**/*` is the safest and is what we need. What's the rationale for changing away from it?

Reply:
Sorry for the confusion!

With the `./**/*` syntax there LiveReload doesn't start up. It _seems_ to be because of the starting `*`. It hangs like the gif at the end.

```
HEAD is now at 11bf350 Tweak code a little
alam@tesla website ((HEAD detached at 11bf350)) $ yarn start
yarn run v1.6.0
warning package.json: No license field
$ node ../lib/start-server.js
```

***

Delete the `.*` syntax locally and everything is fine:

```
alam@tesla website (amy/add-livereload-final) $ git co 2c264e898efda48ccd6bca9ad7e5f8225fce1c82
Note: checking out '2c264e898efda48ccd6bca9ad7e5f8225fce1c82'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 2c264e8 Simplify process.env call
alam@tesla website ((HEAD detached at 2c264e8)) $ yarn start
yarn run v1.6.0
warning package.json: No license field
$ node ../lib/start-server.js
Starting Docusaurus server on http://localhost:3000
Livereload listening on 35729...
```

***

So now how to fix watching all of `website/`? Locally, change to `**/*.*` It _also_ doesn't work.

![docu](https://user-images.githubusercontent.com/1372946/40131487-f67dacd0-58ee-11e8-8830-8be07205ca9b.gif)

Reply:
Even delete docs (which is wrong) and just ```    [
      '**/*', // website
    ],``` doesn't run, like the gif? 

Something else I'm missing? 🤔

Reply:
That's super odd. Is working fine for me. Could you reinstall the dependencies or at worst, fetch a clean copy of the repository?

Reply:
Change it back to `**/*` for now. I'm pretty confident that's the right path to specify.